src/ruby/yast/wfm.rb

```diff
@@ -273,7 +273,7 @@ def self.call_builtin_wrapper(*args)
         end
       else
         Yast.import "Report"
-        Report.Error(msg)
+        Report.LongError(msg.gsub(/\n/, '<br />')
```

Comment:
Well, when I see it, why not use new Yast2::Popup? It has stuff like details, rich text support and more.

Reply:
this is in the end turned into ```Popup.LongError``` but Report adds stuff logging and some other stuff

Reply:
and whats more also special autoyast configuration. So lets keep it as it is.